packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -1346,7 +1398,7 @@ void main() {
 
       await tester.press(find.byKey(key));
       await tester.pump();
-      await tester.pump(const Duration(milliseconds: 1000));
+      await tester.pump(const Duration(milliseconds: 300));
```

Comment:
Just a question, why do we change the duration here?

Reply:
Because it didn't need to be that long, no other reason.